lib/teiserver/player/tachyon_handler.ex

```diff
@@ -140,6 +140,14 @@ defmodule Teiserver.Player.TachyonHandler do
     {:stop, :normal, state}
   end
 
+  def handle_command("system/serverStats" = cmd_id, "request", message_id, _message, state) do
+    user_count = Teiserver.Player.SessionRegistry.get_count()
+
+    resp = Schema.response(cmd_id, message_id, %{userCount: user_count}) |> Jason.encode!()
+
+    {:reply, :ok, {:text, resp}, state}
```

Comment:
That works, but since https://github.com/beyond-all-reason/teiserver/pull/531 there's a shorter way to express that:
```elixir
{:response, cmd_id, %{userCount: user_count}, state}
```
so you don't need to deal with any json encoding by hand.

Reply:
Nice, thanks.